56.md

```diff
@@ -63,10 +63,9 @@ Example events
 {
   "kind": 1984,
   "tags": [
-    ["p", <impersonator pubkey>, "impersonation"],
-    ["p", <victim pubkey>]
```

Comment:
A bit dangerous putting a `p` tag here for clients that don't parse this properly. I imagine it could be easy for a client to misinterpret a victim pubkey for an impersonator pubkey. Removing it from the NIP to reduce the risk of this happening. Instead, favouring just adding it to the `content` field.